packages/x-charts/src/ChartsXAxis/ChartsXAxis.tsx

```diff
@@ -83,6 +83,12 @@ function addLabelDimension(
   });
 }
 
+const XAxisRoot = styled(AxisRoot, {
+  name: 'MuiChartsXAxis',
+  slot: 'Root',
+  overridesResolver: (props, styles) => styles.root,
+})({});
```

Comment:
Looks like a bit of overwork to style a component already styled. But it improve consistency.

I'm considering removing the `AxisRoot` for v8. WHat do you think?

Reply:
What do you mean by removing it? Move the styles to each of the axis?